dropbox/oauth.py

```diff
@@ -297,19 +296,19 @@ def __init__(self, consumer_key, consumer_secret=None, locale=None, token_access
             By default, the server uses "en_US".
         :param str token_access_type: the type of token to be requested.
             From the following enum:
-            legacy - creates one long-lived token with no expiration
-            online - create one short-lived token with an expiration
-            offline - create one short-lived token with an expiration with a refresh token
+            * legacy - creates one long-lived token with no expiration
```

Comment:
Not sure if lists need to only be defined like this or if an empty line needs to be inserted but the current formatting just displays as one line.

This comment also applies to all other lists I added.

Reply:
Confirmed in #228